tools/Azure.Mcp.Tools.LoadTesting/src/Commands/LoadTestResource/TestResourceListCommand.cs

```diff
@@ -19,6 +19,9 @@ public sealed class TestResourceListCommand(ILogger<TestResourceListCommand> log
         $"""
         Fetches the Load Testing resources for the current selected subscription, resource group in the logged in tenant.
         Returns a list of Load Testing resources.
+        Required parameters: --subscription <subscription> --resource-group <resource-group>
+        Example:
+        azmcp loadtesting testresource list --subscription <subscription> --resource-group <resource-group>
```

Comment:
We don't include parameter info in description because all that info is set to the client in parameter description.  Also don't pass it azmcp as it doesn't have that context.